tools/cli/internal/changelog/outputfilter/operationconfig _test.go

```diff
@@ -0,0 +1,303 @@
+package outputfilter
```

Comment:
there is an space in the file name somehow ` _test.go`

Reply:
ops, you are right

Reply:
updated